ludwig/api.py

```diff
@@ -271,6 +271,9 @@ def train(
             data_validation_hdf5=None,
             data_test_hdf5=None,
             data_dict=None,
+            data_train_dict=None,
+            data_test_dict=None,
```

Comment:
can you invert the order here, in the docstring and in the rest of the file? as we always use the convention: train, validation, test ;)

Reply:
sure. makes sense